python/mlx/nn/layers/upsample.py

```diff
@@ -121,8 +180,8 @@ class Upsample(Module):
             If a ``float`` is provided, it is the multiplier for all spatial dimensions.
             Otherwise, the number of scale factors provided must match the
             number of spatial dimensions.
-        mode (str, optional): The upsampling algorithm, either ``"nearest"`` or
-            ``"linear"``. Default: ``"nearest"``.
+        mode (str, optional): The upsampling algorithm, either ``"nearest"``,
```

Comment:
Perfect. Would it be too much to ask you to change the paragraph above as well?

Reply:
Not at all, I should have updated that all at once. Sorry for overlooking the other part of the doc. I just have a new commit.